dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -312,6 +312,11 @@ EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured i
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.explanation=The name and value attributes for a JPA default property must not be empty.
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.useraction=Change the server configuration to provide a non-empty name and value for the property or remove the property.
 
+# 0=property name, 1=property value
+JDBC_PROP_NAME_CWWJP0057W=CWWJP0057W: A JPA default property configured in the server.xml file is not recommended to be used here: name = [{0}], value = [{1}].
+JDBC_PROP_NAME_CWWJP0057W.explanation=This JDBC property is not recommended for EE applications and should only be used in JSE environments.
+JDBC_PROP_NAME_CWWJP0057W.useraction=Change the server/application configuration to use a JNDI data source and remove the property.
```

Comment:
Change the server or application configuration to use a JNDI data source and remove the property.